compiler/rustc_middle/src/ty/instance.rs

```diff
@@ -121,6 +121,13 @@ pub enum InstanceDef<'tcx> {
     /// glue.
     DropGlue(DefId, Option<Ty<'tcx>>),
 
+    // /// Compiler-generated `<T as AsyncDroppable>::AsyncInPlaceDropper::poll` implementation.
+    // ///
+    // /// The `DefId` is for `core::ptr::drop_in_place`.
+    // /// The `Option<Ty<'tcx>>` is either `Some(T)`, or `None` for empty drop
+    // /// glue.
+    // AsyncDropGlue(DefId, Option<Ty<'tcx>>),
+    //
```

Comment:
Is this from a follow-up PR?

Reply:
Oh, it seems like I didn't stash this change with other compiler generated async destructor type for ADTs code.